### PR TITLE
e2e flow fixes

### PR DESCRIPTION
**Pull request by valentinyanakiev**
- Added more logging
- Auto-ack event pattern messages
- Updated client-lib dependency and changed Alkemio client factory - how Kratos public endpoint is initialized

### CODE REVIEW

File: src/utils/validate-connection.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { AlkemioClient } from '@alkemio/client-lib';
+import * as dotenv from 'dotenv';
+
+const main = async () => {
+  dotenv.config();
+
+  const ctClient = new AlkemioClient({
```
Comment (hero101) [inline src/utils/validate-connection.ts:7]: `ctClient` again

**Review by hero101 — CHANGES_REQUESTED** [review]

File: .env.docker [hunk 2]
```diff
@@ -1,2 +1,4 @@
 EMAIL_SMTP_HOST=host.docker.internal
-RABBITMQ_HOST=host.docker.internal
\ No newline at end of file
+RABBITMQ_HOST=host.docker.internal
+ALKEMIO_SERVER_ENDPOINT=http://host.docker.internal/admin/graphql
+SERVICE_ACCOUNT_PASSWORD=rie1@iT0
```
Comment (techsmyth) [inline .env.docker:4]: this should probably be a value such as 'change-me-now'

**Review by techsmyth — CHANGES_REQUESTED** [review]
Agree with comments from Svetoslav; one additional comment.

Also please add the SERVICE_ACCOUNT_USERNAME into the .env.docker file. Related to this please update the default value in the notifications.yml file to **NOT** be admin@alkem.io. We should try to stop using this account.

**Comment by valentinyanakiev** [comment]
> Agree with comments from Svetoslav; one additional comment.
> 
> Also please add the SERVICE_ACCOUNT_USERNAME into the .env.docker file. Related to this please update the default value in the notifications.yml file to **NOT** be [admin@alkem.io](mailto:admin@alkem.io). We should try to stop using this account.

I actually deleted it. No point to have the container of the service that way, as we discussed earlier - will remove quickstart-server from the server repo as well..

File: .env.docker (hunk 2 above)
Comment (valentinyanakiev) [reply .env.docker]: Removed .env.docker - it's no longer needed. Extracted the defaults in constants, as suggested.

**Review by techsmyth — APPROVED** [review]
Looks clean. Only thought might be to rename the quickstart-mailslurper to be quickstart-services to match other repos - but not worth holding this up.

**Review by hero101 — APPROVED** [review]

**Pull request merged**